hack/ensure-kind.sh

```diff
@@ -42,8 +42,8 @@ verify_kind_version() {
   fi
 
   local kind_version
-  kind_version=$(kind version)
```

Comment:
We could probably use `kind version -q` instead?

Reply:
Good idea. Updated the PR.